src/services/CountriesService.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+const BASE_URL = 'https://restcountries.com/v3.1/';
```

Comment:
- [ ] [OPTIONAL] You did great on the project, but the app is not working as expected. I believe the issue is that the API service providing the information is unavailable. I recommend you find another API that provides a better service. 🤝️

<img src="https://user-images.githubusercontent.com/17243325/225875837-805f603b-d0c5-4f8a-90e3-ba0fcc31ae91.png" width="300">